cli/cli.ts

```diff
@@ -1166,6 +1165,10 @@ function uploadCoreAsync(opts: UploadOptions) {
         })
 }
 
+function saveLocalPxTarget(cfg: pxt.TargetBundle) {
+    fs.writeFileSync("pxtarget-temp.json", JSON.stringify(cfg, null, 4));
```

Comment:
Don't do that. The pxtarget.json is already rewritte as part of the build and dump into the built folder as "target.json" and "target.js". Rewriting should occur there.